app/views/api/v1/models/_user.json.jbuilder

```diff
@@ -28,5 +28,6 @@ json.accounts do
     # availability derived from presence
     json.availability_status account_user.availability_status
     json.auto_offline account_user.auto_offline
+    json.features account_user.account.enabled_features
```

Comment:
@muhsin-k I think this change is unnecessary. We use `/api/v1/accounts/{account_id}` to fetch more details about the account on the dashboard.

This is the flow we have 
`validate_token -> fetch_single_account_details -> render dashboard`

Adding more information on the user API call might not be a good option to have.